usaspending_api/download/helpers/download_annotation_functions.py

```diff
@@ -19,11 +19,24 @@
     DecimalField,
     Case,
     When,
+    Q,
 )
 
 AWARD_URL = f"{HOST}/#/award/" if "localhost" in HOST else f"https://{HOST}/#/award/"
 
 
+def filter_by_closed_periods():
```

Comment:
It is probably a better idea to pull the function out of `DisasterBase` and define once in case something needs to change. Using this function with `generate_fiscal_year_and_month()` seems like a solid design to reduce magic numbers in the codebase